src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -105,14 +105,17 @@ class HuiGenericEntityRow extends LitElement {
                     .datetime=${stateObj.last_changed}
                   ></ha-relative-time>
                 `
-              : this.config.secondary_info === "last-triggered" &&
-                stateObj.attributes.last_triggered
-              ? html`
-                  <ha-relative-time
-                    .hass=${this.hass}
-                    .datetime=${stateObj.attributes.last_triggered}
-                  ></ha-relative-time>
-                `
+              : this.config.secondary_info === "last-triggered"
+              ? stateObj.attributes.last_triggered
+                ? html`
+                    <ha-relative-time
+                      .hass=${this.hass}
+                      .datetime=${stateObj.attributes.last_triggered}
+                    ></ha-relative-time>
+                  `
+                : this.hass.localize(
+                    "ui.panel.lovelace.card.entities.never_triggered"
+                  )
               : ""}
```

Comment:
```suggestion
              }
```

Reply:
Hi Ludeeus,

This was already in the original file. Should it be taken out ?

Reply:
No, just looked a little off with all the nesting.